docs/GoogleSubscriber-streamingsource.md

```diff
@@ -28,6 +28,26 @@ It can be found on the Dashboard in the Google Cloud Platform Console.
 
 **Subscription**: Name of the Google Cloud PubSub subscription to subscribe.
 If the subscription needs to be created then the topic to which the subscription will belong must be provided.
+Naming Convention for Subscription:   
+                        
+    Not begin with the string goog.
```

Comment:
Please fix the formatting to look like the Pub/Sub docs page
- Make these bullet points
- `goog` instead of goog.

Reply:
removed period and made list a bullet point list to better match cloud documentation